api/Makefile.am

```diff
@@ -45,7 +45,7 @@ endif
 libtesseract_api_la_SOURCES = baseapi.cpp capi.cpp renderer.cpp pdfrenderer.cpp
 
 lib_LTLIBRARIES += libtesseract.la
-libtesseract_la_LDFLAGS = 
+libtesseract_la_LDFLAGS = $(OPENCL_LDFLAGS)
```

Comment:
Ah, so _this_ is the variable to modify… (instead of one named `local_LDFLAGS` like you initially suspected(?)/suggested in [your earlier comment](https://github.com/tesseract-ocr/tesseract/issues/786#issuecomment-292648687) on #786.)